app_dart/lib/src/foundation/utils.dart

```diff
@@ -179,6 +179,20 @@ Future<List<Target>> getTargetsToRun(Iterable<Target> targets, List<String?> fil
   return targetsToRun;
 }
 
+/// Returns `true` if [ciYaml.postsubmitTargets] should be ran during presubmit.
+///
+/// This is an **experimental** feature that is only enabled for the
+/// `flutter/engine` repository if a specific label is present on the PR.
```

Comment:
Can you move this into `scheduler.dart`?

(1) It's only called from there
(2) Generally, having static globals in an anti-pattern, and we've been working to remove the references here.

Reply:
Done.